x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -30,7 +30,8 @@
 import static org.elasticsearch.xpack.core.security.authz.store.ReservedRolesStore.getRemoteIndicesReadPrivileges;
```

Comment:
Linting has resulted in changes to the comments:
`./gradlew :x-pack:plugin:core:spotlessApply`